configure.self

```diff
@@ -53,10 +53,14 @@ function checks {
     mkl_lib_check "libpthread" "" fail CC "-lpthread" \
                   "#include <pthread.h>"
 
-    if [[ $ENABLE_C11THREADS == "y" ]]; then
+    if [[ $ENABLE_C11THREADS != n ]]; then
+        case "$ENABLE_C11THREADS" in
```

Comment:
Maybe make a helper function, `mkl_ynd "some-string"` that returns `y`, `n` or `d` depending on the string.
E.g.:
```bash
local action=$(mkl_ynd "$ENABLE_C11THREADS")
```